l10n_vn_account_invoice_number/models/account_voucher.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Trobz (http://trobz.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountVoucher(models.Model):
+    _inherit = "account.voucher"
+
+    invoice_number = fields.Integer()
```

Comment:
Could we add the prefix `tax_` to be consistent with the account.invoice?

Reply:
Hi, I'm still not work on the voucher.
I'm still not know the use of the voucher.
is that still the same as invoice?
is that need these 3 field too?
as the previous work of @haidd , I'm confuse that he is hide these 3 field while user choose the "pay now".

Reply:
I don't think we need on voucher because voucher should be used only when we want to record an accounting transaction which is not an invoice, so the extra-fields on voucher do not make sense.

So if you agree with my thoughts and considering that @haidd does not follow-up, I think we should remove all code and documentation related to vouchers.

Reply:
agree, removed.